### PR TITLE
Support for internal display and quick settings submenu

### PR DESCRIPTION
**Pull request by slaclau**
Respond to #107 

- **Add option for individual sliders in quick settings sub menu Sync all slider to average of individual sliders**
- **Add settings items to quick settings sub menu**
- **Add functionality to smoothly vary display sliders when changing all slider**
- **Add slider for internal display**

### CODE REVIEW

File: display-brightness-ddcutil@themightydeity.github.com/extension.js [hunk 1]
```diff
@@ -392,6 +512,17 @@ export default class DDCUtilBrightnessControlExtension extends Extension {
     }
 
     parseDisplaysInfoAndAddToPanel(ddcutilBriefInfo) {
+        if (this.settings.get_boolean('show-internal-slider')) {
+            const _sync = () => {
+
```
Comment (daitj) [inline display-brightness-ddcutil@themightydeity.github.com/extension.js:517]: Empty function? work in progress or?

File: display-brightness-ddcutil@themightydeity.github.com/extension.js [hunk 2]
```diff
@@ -208,26 +225,97 @@ export default class DDCUtilBrightnessControlExtension extends Extension {
...
         reloadItem.connect('activate', event => {
             this.reloadExtension();
         });
-        mainMenuButton.addMenuItem(reloadItem, 2);
+        if (this.settings.get_boolean('show-sliders-in-submenu') && this.settings.get_boolean('show-all-slider')) {
+            mainMenuButton.getStoredSliders()[0].menu.addMenuItem(reloadItem)
+        } else if (this.settings.get_int('button-location') === 0) {
+            mainMenuButton.addMenuItem(reloadItem, 2);
+        }
     }
```
Comment (daitj) [inline display-brightness-ddcutil@themightydeity.github.com/extension.js:319]: Need to flip the condition check or else `mainMenuButton.getStoredSliders()[0]` becomes undefined when  `this.settings.get_boolean('show-sliders-in-submenu') && this.settings.get_boolean('show-all-slider')` is true when  `this.settings.get_int('button-location') === 0` is true as well. Should be like Line 305-309.

Remeber to check that the extension works when the main button is in the top bar instead system menu of once you have all the features ready. It is bit of a hassle but the extension should work with both ways.

**Review by daitj — COMMENTED** [review]
you haven't pushed the latest changes from your branch, then it will be ready for merging.

**Comment by slaclau** [comment]
This commit should fix the comments you made - 1f21bf3071c6a17aad228d85a9f687d49b83d016

**Pull request merged**